autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -447,7 +550,39 @@ def finish_up(self, loss: Dict[str, float], train_loss: Dict[str, float],
         * saving the files for the ensembles_statistics
         * generate output for SMAC
         We use it as the signal handler so we can recycle the code for the
-        normal usecase and when the runsolver kills us here :)"""
+        normal usecase and when the runsolver kills us here :)
+
+        Args:
+            loss (Dict[str, float]):
+                The optimization loss, calculated on the validation set. This will
```

Comment:
Probably we can change the line before `This will` and start the next line with `This will`

Reply:
Sorry for the trouble. Could you elaborate? Also, could you use a [suggestion](http://haacked.com/archive/2019/06/03/suggested-changes/) in github?